test/rubocop/cop/minitest/global_expectations_test.rb

```diff
@@ -162,6 +162,51 @@ class GlobalExpectationsTest < Minitest::Test
         end
       RUBY
     end
+
+    define_method(:"registers_offense_when_using_global_#{matcher}_for_chained_hash_reference") do
```

Comment:
To run a test, the prefix must start with `test_`.

```suggestion
    define_method(:"test_registers_offense_when_using_global_#{matcher}_for_chained_hash_reference") do
```

And, if an appropriate method name exceeds the maximum length of `Layout/LineLength`, this repository allows the maximum length of `Layout/LineLength` of .rubocop_todo.yml to be increased. In that case, I think it will be 120 or less.

Reply:
@koic 

Good catch. It was just a typo when I polish the final formatting of the new specs. Fixed 👍.

Also I increased `Layout/LineLength` in the `.rubocop.yml` file and keep ` .rubocop_todo.yml` unchanged as far as it's an autogenerated file. Please let me know it there is a more convenient way to increase a line max length and to solve the issue.